tests/max_wr_rows.test/runit

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+# Tests max_wr_rows_per_txn tunable
+debug=1
+
+[[ "$debug" == 1 ]] && set -x
+db=$1
+
+if [[ -z "$stage" ]]; then
+    echo "Setting stage to 'default' (set 'stage' var to override)."
+    stage="default"
+fi
+
+if [[ -n "$CLUSTER" ]]; then
```

Comment:
You could export CLUSTER to empty in the Mikefile so the database will come up standalone. For instance,
https://github.com/bloomberg/comdb2/pull/468/files#diff-7091688b541d5294bd463281cf3ae3fdR10

Reply:
Oh .. yeah .. that's a great idea .. tx Rivers!